lib/system/qnx/alloc.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2025, BlackBerry Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
```

Comment:
To update in all source file in this PR
```suggestion
/**
```